docs/release-notes/index.md

```diff
@@ -1,8 +1,45 @@
 # Release notes
 
-```{include} /release-notes/release-latest.md
+(v0.11)=
```

Comment:
the `v` is load-bearing, without it the generated HTML anchors are generic `id40` ones.